13/umbraco-engage/upgrading/version-specific-upgrade-notes.md

```diff
@@ -11,3 +11,23 @@ When upgrading to a new minor or patch version, learn about the changes in the [
 {% endhint %}
 
 ## Breaking changes
+#### v13.2.0
+Introduced Razor Class Library support to serve static files for Engage, removing physical backoffice, views, and assets files from development projects.
+
+While this is not considered a breaking change, it is recommended to remove these folders from your project to avoid any potential conflicts in the future.
```

Comment:
Attempted a rewrite of this one, to shorten the sentence a bit.
Let me know what you think!

```suggestion
While this is not considered a breaking change, it is recommended that such folders be removed from the project to avoid conflicts in the future.
```

Reply:
Looks good to me @sofietoft, thank you!! 😁